pom.xml

```diff
@@ -43,7 +43,7 @@
         <kafka.version>3.6.1</kafka.version>
         <kroxy.extension.version>0.8.1</kroxy.extension.version>
         <log4j.version>2.22.1</log4j.version>
-        <caffeine.version>3.1.8</caffeine.version>
+        <caffeine.version>3.1.8.temporary-redhat-00001</caffeine.version>
```

Comment:
Is this version existing in maven repository?

Reply:
sorry - that was unintentional.